modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.tsx

```diff
@@ -57,7 +57,10 @@ const SELECTOR_TRIGGER = `
 	.source-editor__fixed-text__help[data-title],
 	.upper-tbar [data-title]:not(.lfr-portal-tooltip),
 	.upper-tbar [title]:not(.lfr-portal-tooltip),
-	.upper-tbar [data-restore-title]
+	.upper-tbar [data-restore-title],
+	.lfr-tooltip-scope [title],
+	.lfr-tooltip-scope [data-title],
+	.lfr-tooltip-scope [data-restore-title]
```

Comment:
To prevent bugs with empty `title` and `iframe`, these should be
```
.lfr-tooltip-scope [title]:not(iframe,[title=""]),
.lfr-tooltip-scope [data-title]:not([data-title=""]),
.lfr-tooltip-scope [data-restore-title]:not([data-restore-title=""])
```